content/en/blog/releases/v0.1.1-release-notes.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "V0.1"
```

Comment:
The version in the title should be `V0.1.1` ?

Reply:
thanks for catching - fixed in the latest push